### PR TITLE
Pull Dave Reisner's key and use pacman's keyring

### PR DESCRIPTION
**Pull request by elril0000**
The chimera pkg builds successfully with these changes

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ LABEL contributor="shadowapex@gmail.com"
 COPY rootfs/etc/pacman.conf /etc/pacman.conf
 RUN echo -e "keyserver-options auto-key-retrieve" >> /etc/pacman.d/gnupg/gpg.conf && \
     pacman-key --init && \
+    pacman-key --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 1EB2638FF56C0C53 && \
```
Comment (Samsagax) [inline Dockerfile:6]: Van this be avoided if `auto-key-retrieve` is in effect with the change below?

Comment (elril0000) [reply Dockerfile:6]: I just tested it and I think so. I'll push the change

Comment (Samsagax) [reply Dockerfile]: Great

**Comment by Samsagax** [comment]
Thanks for your contribution

**Pull request merged**